docs/src/main/paradox/scala/http/server-side/server-https-support.md

```diff
@@ -1,22 +1,22 @@
 # Server-Side HTTPS Support
 
-Akka HTTP supports TLS encryption on the server-side as well as on the @ref[client-side](client-side/client-https-support.md).
+Akka HTTP supports TLS encryption on the server-side as well as on the @ref[client-side](../client-side/client-https-support.md).
 
 The central vehicle for configuring encryption is the `HttpsConnectionContext`, which can be created using
 the static method `ConnectionContext.https` which is defined like this:
 
 Scala
-:  @@snip [ConnectionContext.scala](../../../../../../akka-http-core/src/main/scala/akka/http/scaladsl/ConnectionContext.scala) { #https-context-creation }
+:  @@snip [ConnectionContext.scala](../../../../../../../akka-http-core/src/main/scala/akka/http/scaladsl/ConnectionContext.scala) { #https-context-creation }
```

Comment:
We could introduce a variable here like we did for akka at https://github.com/akka/akka/blob/master/akka-docs/build.sbt#L45